src/chttpd/src/chttpd_xframe_options.erl

```diff
@@ -71,7 +71,7 @@ check_host(#httpd{mochi_req = MochiReq} = Req, Config) ->
 
 
 get_xframe_config(#httpd{xframe_config = undefined}) ->
-    EnableXFrame = config:get("httpd", "enable_xframe_options", "false") =:= "true",
+    EnableXFrame = chttpd_util:get_chttpd_config_boolean(enable_xframe_options, false) =:= true,
```

Comment:
Would be just `chttpd_util:get_chttpd_config_boolean(enable_xframe_options, false)` I think?